docs/integrations/index.md

```diff
@@ -6,3 +6,27 @@ nav_order: 20
 has_children: true
 redirect_from: /using
 ---
+
+# Integrations
+
+   **💡 Help improve lakeFS**<br/>
+   Part of lakeFS' mission is to seamlessly integrate with all data frameworks and tools.
+   If there's a integration you're missing or one you believe should be improved, feel free to [discuss on GitHub](https://github.com/treeverse/lakeFS/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fintegrations).
+   {: .note }
+
+## Integrating using the S3 Gateway
```

Comment:
Can we make the language in the title more open to non-AWS users so that we're not scaring them off? e.g. at least "S3-compatible Gateway"